src/games/strategy/triplea/delegate/AbstractPlaceDelegate.java

```diff
@@ -558,9 +558,18 @@ protected String isValidPlacement(final Collection<Unit> units, final Territory
   }
 
   /**
-   * Make sure the player has enough in hand to place the units.
+   * @deprecated Alternate version of htis method exists with fewer parameters, user that. This
```

Comment:
Turns out doing this type of deprecation is not necessary and was just an initial way the params were removed when I started on this. I'm going to close this PR, the deprecation route is not the way to go - unused params should just be removed. The compiler will ensure for us that RMI is not broken.